mobile/src/main/java/org/hawkular/client/android/push/PushClient.java

```diff
@@ -56,7 +59,14 @@ public void setUpPush() {
             return;
         }
 
-        RegistrarManager.config(PushConfiguration.NAME, AeroGearGCMPushConfiguration.class)
+        FirebaseApp.initializeApp(this.context, new FirebaseOptions.Builder()
```

Comment:
I added FirebaseApp.initializeApp so that we do not need to hard code values in app via google-services.json

With this we will later be able to fetch these data from REST endpoint and will let admins use there own UPS with firebase